### ISSUE TITLE
Could see the device as OpenCL 1.2 itself instead of 3.0?

### ISSUE THREAD

**Issue by abilashcgit** [pocl/pocl#1093]
Could see the version being displayed as " OpenCL 1.2 PoCL HSTR: CUDA-sm_61". Is there no OpenCL 3.0 support for Nvidia GPU?
Any specific reason for the same?

$ poclcc -l
[2022-09-07 03:32:20.137023942]POCL: in fn pocl_install_sigfpe_handler at line 229:
  |   GENERAL |  Installing SIGFPE handler...
[2022-09-07 03:32:20.159933606]POCL: in fn pocl_cuda_init at line 388:
  |   GENERAL |  [CUDA] GPU architecture = sm_61
[2022-09-07 03:32:20.217164808]POCL: in fn POclCreateCommandQueue at line 47:
  |   GENERAL |  Create Command queue on device 0
LIST OF DEVICES:
0:
  Vendor:   NVIDIA Corporation
    Name:   NVIDIA GeForce GTX 1080
 **Version:   OpenCL 1.2 PoCL HSTR: CUDA-sm_61**                <=====================

**Comment by abilashcgit** [comment]
any update on this? Could you please check this query and revert back?

**Comment by matthiasdiener** [comment]
As far as I know, only the pthreads driver supports 3.0, the other drivers support 1.2. For Cuda, SVM support is missing for example to support higher versions of the OpenCL standard (see https://github.com/pocl/pocl/pull/1067).

**Comment by pjaaskel** [comment]
That's right, although SVM is not needed for 3.0 minimal feature set, IIRC.

**Comment by matthiasdiener** [comment]
If I understood it correctly (but I might be wrong), at least coarse-grained SVM is required by 3.0: https://registry.khronos.org/OpenCL/specs/3.0-unified/pdf/OpenCL_API.pdf (page 35)

**Comment by pjaaskel** [comment]
Seems you are right! I somehow have assumed SVM is too "intrusive" core feature.

**Comment by pjaaskel** [comment]
@matthiasdiener  seems CG SVM is _not_ required afterall by the minimal OpenCL 3.0 spec.
